# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/cocoa/nsMacFinderProgress.h
# Commit: 8287c3d0c106
# Full Hash: 8287c3d0c1064096ee7c162060e6615d72722ee0
# Author: Christoph Walcher <christoph-wa@gmx.de>
# Date: 2019-06-20 09:46:31
# Regressor Bug: 909760
# File Overlap Count: 1
# Description:
#   Bug 909760 - Show download progress in the MacOS Finder r=spohl,mak
#   
#   Show download progress in the MacOS Finder
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D30688
# ==============================================================================

diff -r 5b9a3de04646 -r 8287c3d0c106 widget/cocoa/nsMacFinderProgress.h
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/widget/cocoa/nsMacFinderProgress.h	Wed Jun 19 18:48:49 2019 +0000
@@ -0,0 +1,24 @@
+/* -*- Mode: c++; tab-width: 2; indent-tabs-mode: nil; -*- */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+#ifndef _MACFINDERPROGRESS_H_
+#define _MACFINDERPROGRESS_H_
+
+#include "nsIMacFinderProgress.h"
+#include "nsCOMPtr.h"
+
+class nsMacFinderProgress : public nsIMacFinderProgress {
+ public:
+  nsMacFinderProgress();
+
+  NS_DECL_ISUPPORTS
+  NS_DECL_NSIMACFINDERPROGRESS
+
+ protected:
+  virtual ~nsMacFinderProgress();
+
+  NSProgress* mProgress;
+};
+
+#endif